tests/actions/test_log_event.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+import unittest
+
+import csv
+from rr.actions.log_event import LogEvent, LogEventError, log, validate_csv, set_file_headers, find_max_probability, log
+
+
+class mockMedia():
+    def __init__(self, name):
+        self._name = name
+
+    def get_name(self):
+        return self._name
+
+
+class mockImage():
+    def __init__(self, timestamp):
+        self._timestamp = timestamp
+
+    def get_timestamp(self):
+        return self._timestamp
+
+
+class TestLogEventSuccess(unittest.TestCase):
+
+    def setUp(self):
+        open('tests/actions/test_log.csv', 'w').close()
```

Comment:
Why do we have to commit an empty file? Does your class not work if the file does exist prior?